stdlib/sampledata/sampledata.flux

```diff
@@ -0,0 +1,310 @@
+package sampledata
+
+
+import "csv"
+
+option start = 2021-01-01T00:00:00Z
+option stop = 2021-01-01T00:01:00Z
+
+_numeric = (includeNull=false) => "
+#group,false,false,false,true,false
+#datatype,string,long,dateTime:RFC3339,string,double
+#default,_result,,,,
+,result,table,_time,tag,_value
+,,0,2021-01-01T00:00:00Z,t1,-2.18
+,,0,2021-01-01T00:00:10Z,t1," + (if includeNull then "" else "10.92") + "
+,,0,2021-01-01T00:00:20Z,t1,7.35
+,,0,2021-01-01T00:00:30Z,t1," + (if includeNull then "" else "17.53") + "
+,,0,2021-01-01T00:00:40Z,t1," + (if includeNull then "" else "15.23") + "
+,,0,2021-01-01T00:00:50Z,t1,4.43
+,,1,2021-01-01T00:00:00Z,t2," + (if includeNull then "" else "19.85") + "
+,,1,2021-01-01T00:00:10Z,t2,4.97
+,,1,2021-01-01T00:00:20Z,t2,-3.75
+,,1,2021-01-01T00:00:30Z,t2,19.77
+,,1,2021-01-01T00:00:40Z,t2," + (if includeNull then "" else "13.86") + "
+,,1,2021-01-01T00:00:50Z,t2,1.86
+"
+
+_string = (includeNull=false) => "
+#group,false,false,false,true,false
+#datatype,string,long,dateTime:RFC3339,string,string
+#default,_result,,,,
+,result,table,_time,tag,_value
+,,0,2021-01-01T00:00:00Z,t1,smpl_g9qczs
+,,0,2021-01-01T00:00:10Z,t1," + (if includeNull then "" else "smpl_0mgv9n") + "
+,,0,2021-01-01T00:00:20Z,t1,smpl_phw664
+,,0,2021-01-01T00:00:30Z,t1," + (if includeNull then "" else "smpl_guvzy4") + "
+,,0,2021-01-01T00:00:40Z,t1," + (if includeNull then "" else "smpl_5v3cce") + "
+,,0,2021-01-01T00:00:50Z,t1,smpl_s9fmgy
+,,1,2021-01-01T00:00:00Z,t2," + (if includeNull then "" else "smpl_b5eida") + "
+,,1,2021-01-01T00:00:10Z,t2,smpl_eu4oxp
+,,1,2021-01-01T00:00:20Z,t2,smpl_5g7tz4
+,,1,2021-01-01T00:00:30Z,t2,smpl_sox1ut
+,,1,2021-01-01T00:00:40Z,t2," + (if includeNull then "" else "smpl_wfm757") + "
+,,1,2021-01-01T00:00:50Z,t2,smpl_dtn2bv
+"
+
+_bool = (includeNull=false) => "#group,false,false,false,true,false
+#datatype,string,long,dateTime:RFC3339,string,boolean
+#default,_result,,,,
+,result,table,_time,tag,_value
+,,0,2021-01-01T00:00:00Z,t1,true
+,,0,2021-01-01T00:00:10Z,t1," + (if includeNull then "" else "true") + "
+,,0,2021-01-01T00:00:20Z,t1,false
+,,0,2021-01-01T00:00:30Z,t1," + (if includeNull then "" else "true") + "
+,,0,2021-01-01T00:00:40Z,t1," + (if includeNull then "" else "false") + "
+,,0,2021-01-01T00:00:50Z,t1,false
+,,1,2021-01-01T00:00:00Z,t2," + (if includeNull then "" else "false") + "
+,,1,2021-01-01T00:00:10Z,t2,true
+,,1,2021-01-01T00:00:20Z,t2,false
+,,1,2021-01-01T00:00:30Z,t2,true
+,,1,2021-01-01T00:00:40Z,t2," + (if includeNull then "" else "true") + "
+,,1,2021-01-01T00:00:50Z,t2,false
+"
+
+// float returns a sample data set with float values.
+//
+// ## Parameters
+//
+// - `includeNull` indicates whether or not to include null values in the returned dataset.
+//   Default is `false`.
+//
+// ## Output basic sample data with float values
+//
+// ```
+// import "sampledata"
+//
+// sampledata.float()
+// ```
+// 
+// ## Output data
```

Comment:
What is the motivation for including the output in the doc comment directly? Would it be easy enough to just run the example to see the data? Having the data in the comment means that if it changes we have to update the doc. Also it doesn't show what the data looks like with nulls so to see that you would have to just run the function. Thoughts? The data shouldn't change much but still wanted to ask about it.

Reply:
Visibility. Just a way to see the sample data whenever the docs are generated from the stdlib.